src/server/models/okta/User.ts

```diff
@@ -101,3 +101,12 @@ export enum Status {
   PASSWORD_RESET = 'PASSWORD_RESET',
   SUSPENDED = 'SUSPENDED',
 }
+
+// https://developer.okta.com/docs/reference/api/sessions/#session-object
+export interface SessionResponse {
```

Comment:
Is this is already defined here https://github.com/guardian/gateway/blob/50c35f0a6a42129d68acd3f58a1b6d0d1db3f850/src/server/models/okta/Session.ts

Reply:
Yes it is, I'll remove the unnecessary import!